pulsar-broker/src/main/java/org/apache/pulsar/common/naming/ConsistentHashingTopicBundleAssigner.java

```diff
@@ -27,10 +27,9 @@ public class ConsistentHashingTopicBundleAssigner implements TopicBundleAssignme
     private NamespaceService namespaceService;
     @Override
     public NamespaceBundle findBundle(TopicName topicName, NamespaceBundles namespaceBundles) {
-        NamespaceName namespaceName = topicName.getNamespaceObject();
-        checkArgument(namespaceName.equals(topicName.getNamespaceObject()));
         long hashCode = namespaceService.getNamespaceBundleFactory().getLongHashCode(topicName.toString());
         NamespaceBundle bundle = namespaceBundles.getBundle(hashCode);
+        checkArgument(bundle.getNamespaceObject().equals(topicName.getNamespaceObject()));
```

Comment:
It may cause NPE, we should find a way to be compatible with the previous logic in other ways